### ISSUE TITLE
Marvin remote access stalls

### ISSUE THREAD

**Issue by mmezcua** [sdss/marvin#685]
**Describe the bug**
Marvin is not able to access remotely more than ~50ish maps when doing:
>maps = Maps(plateifu=i)
where i is a list of MaNGA IDs. The script works well for the first 50ish IDs. 

**Expected behaviour**
I was expecting the script to go through all the IDs. I have around 430.

**Screenshots**
MarvinError: found a problem when getting the map: Requests Http Status Error: 429 Client Error: TOO MANY REQUESTS for url: https://dr15.sdss.org/marvin/api/maps/8239-9102/HYB10/GAU-MILESHC/map/emline_gflux/nii_6585/
Error accessing https://dr15.sdss.org/marvin/api/maps/8239-9102/HYB10/GAU-MILESHC/map/emline_gflux/nii_6585/: 429-Rate Limit Exceeded.

**Desktop (please complete the following information):**
 - OS
 - Browser: safari
 - Version of Marvin: 2.3.0

**Additional context**
Using a script in python 3.5

**Comment by havok2063** [comment]
Hi @mmezcua, this is occurring because we originally imposed a rate limit on our API to prevent spamming.  We chose a reasonable but somewhat arbitrary rate limit of 400 API requests per user per minute.  Each `Map` request makes a few API calls.  For loops over lots of `Maps`, you will eventually hit this limit.  We have decided to remove our rate limits in an upcoming release, because a lot of people have hit them and its more of a hindrance.  This update will be released soon, probably within the next month or so.  Until then one option is to break up your loop into smaller ones or space out the loop over several minutes.  

Related to #685, #502
